src/components/UpcomingEventsList.tsx

```diff
@@ -2,14 +2,13 @@ import { useEffect, useState } from "react"
 import _ from "lodash"
 import { useRouter } from "next/router"
 import { useTranslation } from "next-i18next"
-import { Box, Grid, Heading } from "@chakra-ui/react"
 
 import type { CommunityConference, Lang } from "@/lib/types"
 
-import { Button } from "@/components/Buttons"
 import EventCard from "@/components/EventCard"
 import InfoBanner from "@/components/InfoBanner"
```

Comment:
@pettinarip I see you used [the Alert here](https://github.com/ethereum/ethereum-org-website/pull/13597/commits/bee765c69acbead6caebe339e5502bb915461709) for InfoBanner migration. InfoBanner is used over a thousand of times in our markdown files, so it is probably best we just migrate the component itself. What do you think?

Reply:
Correct, thats the plan for InfoBanner. We are going to tackle that in a differnt PR